js/buttonConnection.js

```diff
@@ -0,0 +1,13 @@
+var currencyChoiceContainer = document.getElementById('currency-choice-container');
+var currentCurrencyButton = document.getElementById('current-currency');
+
+currencyChoiceContainer.onclick = function(e){
+  // var target = e.target;
+  if(usd.checked){
+    currentCurrencyButton.style.backgroundImage="url('img/usd_checked.svg')";
+  } else if(euro.checked){
+    currentCurrencyButton.style.backgroundImage="url('img/euro_checked.svg')";
+  } else if (rrubles.checked) {
+    currentCurrencyButton.style.backgroundImage="url('img/rruble_checked.svg')";
+  }
+};
```

Comment:
- [x]  I've excluded kind of 'usd.checked' appeals from the buttonConnection.js . Nove an ONCLICK function defines an appropriate DOM-object and changes the button background. A new code you can check in [buttonConnection.js]
(https://github.com/IlyaRucavitcyn/vanga.by/blob/buttonConnection/js/buttonConnection.js).

-  [x] The representation and logic layers were spread. To style button in accordance to the checked currency I've created ".USD", ".RUB" and ".EUR" classes with styles defined in [style.css](https://github.com/IlyaRucavitcyn/vanga.by/blob/buttonConnection/css/style.css)(line 169 - 180). Classes are added to the button with the help of [buttonConnection.js](https://github.com/IlyaRucavitcyn/vanga.by/blob/buttonConnection/js/buttonConnection.js).

- [x] All the currency names mentiooned in code are changed in accordance with ISO.

[The latest webview of the page](http://ilyarucavitcyn.github.io/vanga.by/)